tests/unit/jobs/test_healthchecks.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+from githubkit.rest.models import AuthenticationToken
+from hypothesis import given, settings
+from hypothesis import strategies as st
+from redis_om import Migrator
+
+from runner_manager.clients.github import GitHub
+from runner_manager.models.runner import Runner, RunnerStatus
+from runner_manager.models.runner_group import RunnerGroup
+from runner_manager.models.settings import Settings
+
+from ...strategies import GithubRunnerStrategy, SettingsStrategy
+
+
+@given(
+    github_runners=st.lists(GithubRunnerStrategy, min_size=1),
+    settings=SettingsStrategy,
+)
+@settings(max_examples=10)
+def test_healthchecks_hypothesis(
+    github_runners: list, settings: Settings, runner_group: RunnerGroup, github: GitHub
+):
+    # create github runners
+    for github_runner in github_runners:
+        print(github_runner.name)
+        runner = Runner(
+            id=github_runner.id,
+            name=f"runner_{github_runner.id}",
+            status=github_runner.status,
+            busy=github_runner.busy,
+            runner_group_name=runner_group.name,
+            runner_group_id=runner_group.id,
+        )
+        # TODO: might be missing some sort of started at if runner.status is online
```

Comment:
I'll let you solve this todo as you wish, we can let the code as is, it's fine or add some started_at. Same is true for created_at.

Valid example of implicitly creating created_at
```python
# runner_manager/models/runner.py

    def save(
        self,
        pipeline: Optional[redis.client.Pipeline] = None,
    ) -> "Runner":
        """Create a runner.

        Returns:
            Runner: Runner instance.
        """
        if self.created_at is None:
            self.created_at = datetime.now()
        return super().save(pipeline=pipeline)
```